hwdb.d/60-keyboard.hwdb

```diff
@@ -1820,24 +1821,24 @@ evdev:input:b0003v04FEp000D*
  KEYBOARD_LED_CAPSLOCK=0
 
 # Lenovo ThinkPad T430s
-evdev:atkbd:dmi:bvn*:bvr*:bd*:svnLENOVO:pn*:pvrThinkPadT430s
+evdev:atkbd:dmi:bvn*:bvr*:bd*:svnLENOVO:pn*:pvrThinkPadT430s*
  KEYBOARD_LED_CAPSLOCK=0
 
 # Lenovo ThinkPad T440s
-evdev:atkbd:dmi:bvn*:bvr*:bd*:svnLENOVO:pn*:pvrThinkPadT440s
+evdev:atkbd:dmi:bvn*:bvr*:bd*:svnLENOVO:pn*:pvrThinkPadT440s*
  KEYBOARD_LED_CAPSLOCK=0
 
 # Lenovo ThinkPad T450s
-evdev:atkbd:dmi:bvn*:bvr*:bd*:svnLENOVO:pn*:pvrThinkPadT450s
+evdev:atkbd:dmi:bvn*:bvr*:bd*:svnLENOVO:pn*:pvrThinkPadT450s*
  KEYBOARD_LED_CAPSLOCK=0
 
 # Lenovo ThinkPad T560s
-evdev:atkbd:dmi:bvn*:bvr*:bd*:svnLENOVO:pn*:pvrThinkPadT560s
+evdev:atkbd:dmi:bvn*:bvr*:bd*:svnLENOVO:pn*:pvrThinkPadT560s*
  KEYBOARD_LED_CAPSLOCK=0
  KEYBOARD_LED_NUMLOCK=0
 
 # Lenovo ThinkPad X1 Carbon 3rd Gen
-evdev:atkbd:dmi:bvn*:bvr*:bd*:svnLENOVO:pn*:pvrThinkPadX1Carbon3rd
+evdev:atkbd:dmi:bvn*:bvr*:bd*:svnLENOVO:pn*:pvrThinkPadX1Carbon3rd*
```

Comment:
these four too. Plenty others that match evdev: and end in a pvr check do place the `:*`at the end, and I think we should do here too